app/models/submission.rb

```diff
@@ -405,17 +405,22 @@ def proquest_agreement=(input)
 
   private
 
-    def format_review_file_check
+    def file_check
       # no validation for admin users
       return true unless author_edit
-      # only require file when author submitting format review
-      return true unless status_behavior.collecting_format_review_files?
-
-      if format_review_files.nil? || format_review_files.blank?
-        # errors[] << 'You must upload a format review file.'
-        false
-      else
-        true
+
+      if status_behavior.collecting_format_review_files?
+        return true unless format_review_files.nil? || format_review_files.blank?
+
+        errors.add(:format_review_file, "You must upload a Format Review file.")
+
+      elsif status_behavior.collecting_final_submission_files?
+        return true unless final_submission_files.nil? || final_submission_files.blank?
```

Comment:
Doesn't blank include nil? Is the OR needed here & above?

Reply:
Oh right, yes.  `blank?` should be enough here.